node/node.go

```diff
@@ -955,11 +952,21 @@ func NewSeedNode(config *cfg.Config,
 		return nil, fmt.Errorf("failed to create router: %w", err)
 	}
 
-	// start the pex reactor
-	pexReactor := createPEXReactorAndAddToSwitch(addrBook, config, sw, logger)
-	pexReactorV2, err := createPEXReactorV2(config, logger, peerManager, router)
-	if err != nil {
-		return nil, err
+	var (
+		pexReactor   *pex.Reactor
+		pexReactorV2 *pex.ReactorV2
+	)
+
+	// add the pex reactor
+	router.AddChannelDescriptors(pex.ChannelDescriptors)
+	transport.AddChannelDescriptors(pex.ChannelDescriptors)
```

Comment:
Even though this might be temporary, can we add a comment as to _why_ we're doing this?

Reply:
Yup I'll add one